### PR TITLE
logistics and nav order

### PR DESCRIPTION
**Pull request by sangmichaelxie**
Logistics section of index and nav bar ordering.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -29,7 +29,16 @@ well as gain hands-on experience working with them.
 
 ## Logistics
 
-TODO: Where/when
+**Where**: In-person lectures and discussion will be at 200-002 (History Corner). Remote lectures will be on Zoom (Zoom link in the Google Calendar event below). The first two weeks of class will be remote (subject to change according to University policies).
```
Comment (percyliang) [inline index.md]: Can you link to a map?

File: index.md [hunk 2]
```diff
@@ -29,7 +29,16 @@ well as gain hands-on experience working with them.
...
+
+**When**: Mondays and Wednesdays 3:15-4:45pm PST [\[Google Calendar\]](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGUwdHFqdDAycmhicmZhZTBvdDh1bWg0YnNfMjAyMjAxMDNUMjMxNTAwWiBzeGllQHN0YW5mb3JkLmVkdQ&tmsrc=sxie%40stanford.edu&scp=ALL)
```
Comment (percyliang) [inline index.md]: We definitely don't want this public and have anyone being able to join the zoom.

File: index.md [hunk 3]
```diff
@@ -29,7 +29,16 @@ well as gain hands-on experience working with them.
...
+
+**Links**:
+- [Gradescope](https://www.gradescope.com/courses/342794)
```
Comment (percyliang) [inline index.md]: More English description of what we use these for (see other course pages)

**Comment by rishibommasani** [comment]
@percyliang 
Grading, etc. stuff is in https://github.com/stanford-cs324/winter2022/pull/1/commits/4843e3c54a577dfe5161b2967e739a2ab09f9fec and closely mirror the 221 stuff. 
TBDs reflect need to decide on final project

File: index.md [hunk 4]
```diff
@@ -24,18 +24,42 @@ systems across a wide range of tasks and have shown an impressive ability to
...
 
 ## Grading
 
-Two assignments + final project
+- **Paper reviews and class participation** (20%): For each lecture, 1-2 papers will be assigned that should be read in advance. For each lecture, a brief review of the papers should be submitted prior to lecture (that will be graded pass/fail) and students should participate in the discussion of the papers.  
+- **Assignments** (2 x 25%): There will be two assignments (A1 and A2). Assignments should be done in groups of TBD students. Deadlines and details will be released soon.    
+- **Final project** (30%): Similar to other AI courses, this course will have a final project where you use the tools from class to build something interesting of your choice. Projects should be done in groups of TBD students. Deadlines and details will be released soon.
```
Comment (percyliang) [inline index.md]: Update based on discussion.  For TBD put 2-3.

File: index.md [hunk 5]
```diff
@@ -24,18 +24,42 @@ systems across a wide range of tasks and have shown an impressive ability to
...
+- **Assignments** (2 x 25%): There will be two assignments (A1 and A2). Assignments should be done in groups of TBD students. Deadlines and details will be released soon.    
+- **Final project** (30%): Similar to other AI courses, this course will have a final project where you use the tools from class to build something interesting of your choice. Projects should be done in groups of TBD students. Deadlines and details will be released soon.  
+
+## Homeworks
+
+**Paper reviews**: Paper reviews are due at **3:00 PM Pacific Time on [Gradescope](https://www.gradescope.com/courses/342794)** on the day of the lecture. Reviews should be no more than two pages and should address: (i) the main contributions of the paper(s), (ii) the strengths and weaknesses of the paper, and (iii) questions/discussion items you have based on the work. For each lecture, a few students will be designated (in advance) to provide the initial questions/discussion items for the lecture's discussion based on their responses in (iii).
```
Comment (percyliang) [inline index.md]: Two pages is a lot...I'd leave this underspecified and provide an example.

**Review by percyliang — APPROVED** [review]

**Pull request merged**